src/ui/shared/BrutalistUI.jsx

```
@@ -0,0 +1,929 @@
+import React, { useState, useEffect, useRef } from 'react';
+
+// --- SVG DECORATIONS ---
+
+export const HexBorder = ({ className }) => (
+  <svg className={className} viewBox="0 0 100 100" preserveAspectRatio="none" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Biome (2.4.4)

[error] Alternative text title element cannot be empty

(lint/a11y/noSvgWithoutTitle)